/**:
  ros__parameters:
    # controller type: 'lead_lag', 'pid', or 'none'
    controller_type: 'none'
    # parameters for all controllers
    limits:
      throttle:
        min: -1.5
        max:  1.5
    # parameters for none (open loop)
    open_loop:
      publish_early: true
    # parameters for lead_lag (stable with 0.2 sec delay)
    # lead:
    #   gain:   2.2696
    #   alpha: -0.6146
    #   beta:  -0.7130
    # lag:
    #   alpha: -0.9608
    #   beta:  -0.9951
    # alternate lead_lag (with predictive control)
    lead:
      gain:   3.9229
      alpha: -0.2584
      beta:  -0.9231
    lag:
      alpha: -0.573
      beta:  -0.9786
    # parameters for pid
    pid:
      kp: 1.0
      ki: 0.0
      kd: 0.0
      anti_windup: 0.0
      deadband: 0.0